config/proto/service.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package hermes;
+
+import "github.com/googleinterns/step224-2020/config/proto/probe.proto";
+import "github.com/googleinterns/step224-2020/config/proto/targets.proto";
+
+option go_package = "github.com/googleinterns/step224-2020/config/proto";
+
+service Hermes {
+
+  // Start monitoring a new storage system.
+  rpc StartMonitoringStorageSystem(HermesProbeRequest) returns (HermesProbeResponse) {}
+
+  rpc StopMonitoringStorageSystem(StopMonitoringSystemRequest) returns (StopMonitoringSystemResponse) {}
```

Comment:
nit: Despite the name of the RPC being pretty self explanatory, please add a comment in the line right above.

Reply:
```suggestion
  // Stop monitoring a storage system that is currently being monitored.
  rpc StopMonitoringStorageSystem(StopMonitoringSystemRequest) returns (StopMonitoringSystemResponse) {}
```